tools/profiletool_core.py

```diff
@@ -30,6 +30,7 @@
 
 # qgis import
 from qgis.core import QgsGeometry, QgsMapLayer, QgsPoint, QgsPointXY, QgsProject
+from qgis.core import QgsVectorLayer, QgsFeature # added by C.Candido
```

Comment:
```suggestion
from qgis.core import QgsVectorLayer, QgsFeature
```

No need, git blame does this for us and it overloads the code.
Could remove all others ?

Reply:
This was just internally for me to track the code changes. I have removed these comments.